src/main/scala/encry/api/http/routes/ConfigRoute.scala

```diff
@@ -0,0 +1,472 @@
+              )
+            )
+            ,
+            // Footer
+            footer(cls := "footer",
+              div(cls := "row align-items-center justify-content-xl-between",
+                div(cls := "col-xl-6",
+                  div(cls := "copyright text-center text-xl-left text-muted", "© 2018",
+                    a(href := "https://www.creative-tim.com", cls := "font-weight-bold ml-1", target := "_blank", "Creative Tim")
+                  )
+                ),
+                div(cls := "col-xl-6",
+                  ul(cls := "nav nav-footer justify-content-center justify-content-xl-end",
+                    li(cls := "nav-item",
+                      a(href := "https://www.creative-tim.com", cls := "nav-link", target := "_blank", "Creative Tim")
+                    ),
+                    li(cls := "nav-item",
+                      a(href := "https://www.creative-tim.com/presentation", cls := "nav-link", target := "_blank", "About Us")
+                    ),
+                    li(cls := "nav-item",
+                      a(href := "http://blog.creative-tim.com", cls := "nav-link", target := "_blank", "Blog")
+                    ),
+                    li(cls := "nav-item",
+                      a(href := "https://github.com/creativetimofficial/argon-dashboard/blob/master/LICENSE.md", cls := "nav-link", target := "_blank", "MIT License")
+                    )
+                  )
+                )
+              )
+            )
+          ),
+        ),
+        // Argon Scripts
+        // Core
+        script(src := "argon/assets/vendor/jquery/dist/jquery.min.js"),
+        script(src := "argon/assets/vendor/bootstrap/dist/js/bootstrap.bundle.min.js"),
+        // Optional JS
+        script(src := "argon/assets/vendor/chart.js/dist/Chart.min.js"),
+        script(src := "argon/assets/vendor/chart.js/dist/Chart.extension.js"),
+        Seq(
+          link(rel := "stylesheet", tpe := "text/css", href := "https://cdn.datatables.net/1.10.20/css/jquery.dataTables.min.css"),
+          script(tpe := "text/javascript", src := "https://cdn.datatables.net/1.10.20/js/jquery.dataTables.min.js")
+        ),
+        // Argon JS
+        script(src := "argon/assets/js/argon.js?v=1.0.0")
+
+      )
+    )
+  }
+
+  def sendAllInfo: Route = (path("collect") & get) {
+    parameters('password, 'mnem, 'chain, 'sync, 'host, 'peer, 'cwp, 'nodePass, 'nodeName, 'declared, 'bind) {
+      (password, mnemonic, chain, sync, host, peer, cwp, nodePass, nodeName, declaredAddr, bindAddr) =>
+
+        val passwordR: String = password
+        val mnemonicR: String = mnemonic match {
+          case x: String => x
+        }
+        val chainR: Boolean = chain match {
+          case "Yes" => true
+          case "No" => false
+        }
+        val syncR: Boolean = sync match {
+          case "Fast" => true
+          case "Normal" => false
+        }
+        val peerR: InetSocketAddress = new InetSocketAddress(host, peer.toInt)
```

Comment:
are you sure that peer.toInt doesn't throw an exception if an input is incorrect